shared/locales/en/website-donate.json

```diff
@@ -1,12 +1,16 @@
 {
 	"metadata": {
-		"title": "My Contribution | Social Income"
+		"title": "Rebuilding Lives With Ismatu",
+		"description": "Donate to support Ebola survivors in Sierra Leone.",
+		"og-image": "/assets/metadata/og/campaign-ismatu-en.png",
```

Comment:
why do we need to pictures here? Is there a difference between `/assets/metadata/twitter/campaign-ismatu-en.png.jpg` and `/assets/metadata/og/campaign-ismatu-en.png`?

Reply:
The recommended size and aspect radio is different for ogImages and Twitter preview images.

Reply:
I see, thanks!